app/app.go

```diff
@@ -1002,6 +1002,8 @@ func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino
 	paramsKeeper.Subspace(icacallbacksmoduletypes.ModuleName)
 	// this line is used by starport scaffolding # stargate/app/paramSubspace
 
+	paramsKeeper.Subspace(claimtypes.ModuleName)
+	paramsKeeper.Subspace(authz.ModuleName)
```

Comment:
Why is this required?

Reply:
Need it to initialize params store for modules

Reply:
Unlike other data, module params are stored differently. The prefixed subspace of the parameter store is manage by `params` module. Some modules need their params private for another, we can modifier it

Reply:
Got it, thanks for the explanation!